packages/dbus-broker/Cargo.toml

```
@@ -9,12 +9,11 @@ build = "build.rs"
 path = "pkg.rs"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/bus1/dbus-broker/releases/download/v23/dbus-broker-23.tar.xz"
-sha512 = "b9431a9733342d4b6214a6a35ad05d3278ab2c8cfdc35537dba86ba9ee3f59eef4e31687c2fe8a6444072eaf8384bc71a41617fec0539a8c00be9eae0efa9832"
+url = "https://github.com/bus1/dbus-broker/releases/download/v23/dbus-broker-24.tar.xz"
```

```suggestion
url = "https://github.com/bus1/dbus-broker/releases/download/v24/dbus-broker-24.tar.xz"
```
Otherwise this actually downloads 23 (eek) and has the wrong checksum.

---

😞